vllm/model_executor/layers/quantization/kernels/__init__.py

```diff
@@ -15,6 +17,7 @@
 _POSSIBLE_KERNELS: List[Type[MPLinearKernel]] = [
     MacheteLinearKernel,
     MarlinLinearKernel,
+    BitBLASLinearKernel,
```

Comment:
is this disabled somehow if bitblas isn't installed? (I thought it might be via `can_implement` but I cant seem to find that check anywhere)

Reply:
related code has been implemented to checkout whether bitblas is installed within can_implement.